spec/awful/completion_spec.lua

```diff
@@ -15,6 +15,37 @@ if not has_bash and not has_zsh then
     return
 end
 
+local orig_path_env = GLib.getenv("PATH")
+local required_commands = {"bash", "zsh", "sort"}
+
+-- Change PATH
+local function get_test_path_dir()
+    local test_path = Gio.File.new_tmp("awesome-tests-path-XXXXXX")
+    test_path:delete()
+    test_path = test_path:get_path()
+    gfs.make_directories(test_path)
+
+    for _, cmd in ipairs(required_commands) do
+        local target = GLib.find_program_in_path(cmd)
+        if target then
+            local link = string.format('%s/%s', test_path, cmd)
+            Gio.File.new_for_path(link):make_symbolic_link(target)
```

Comment:
`target` is a string, not a GFile object, hence there is no `get_child()`. Shall I instantiate `target` as a new GFile object, then?